proto/storage/alert.proto

```diff
@@ -103,7 +103,7 @@ message Alert {
     string message = 2;
   }
 
-  string id = 1; // Internal use only // @gotags: search:"Alert ID,hidden" sensorhash:"ignore" sql:"pk,type(uuid)"
+  string id = 1; // @gotags: search:"Alert ID" sensorhash:"ignore" sql:"pk,type(uuid)"
```

Comment:
Does anyone know why this was "Internal use only"?  Curious if there was a reason.

Reply:
Me too. @clickboo & @mtodor didn't had context, @charmik-redhat could know it. Otherwise I think we can add it, the alert ID is something we expose in our URIs anyway, so nothing gets leaked or so.

Reply:
yeah Its just a UUID.  Alert is scoped to namespaces as far as SAC goes.  Seems like cluster and namespace are in the notification.  So this makes sense to me.

Reply:
I don't remember why that was `Internal use only` . But I don't see any issue making it public.